health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -1286,6 +1356,17 @@ function UploadDataMapping({ formData, onSelect, currentCategories }) {
           footerChildren={[
+            <Button
+              type={"button"}
+              size={"large"}
+              variation={"secondary"}
+              label={t("NO")}
+              onClick={() => {
+                setShowPopUp(false);
+                setSelectedLevel(null);
+                setSelectedBoundary(null);
+              }}
+            />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)